test/setup/deployers/operator.ts

```diff
@@ -1,19 +1,53 @@
+import * as sleep from 'sleep-promise';
 import { IDeployer, IImageOptions } from './types';
 import * as kubectl from '../../helpers/kubectl';
 
+const PULL_KUBERNETES_OPERATOR_EVENT = 'docker.io/snyk/kubernetes-operator';
 export const operatorDeployer: IDeployer = {
   deploy: deployKubernetesMonitor,
 };
 
+async function seekEvent(event: string): Promise<boolean> {
+  const events = await kubectl.getEvents('marketplace');
```

Comment:
Function name looks generic but we use a hardcoded OLM namespace, could we explicitly also pass the namespace to `seekEvent()`?